backend/Messenger/Messenger.Data/Seeder/DbSeeder.cs

```diff
@@ -1,79 +1,64 @@
 using Messenger.Core.Model;
 using Messenger.Core.Model.UserAggregate;
 using Messenger.Core.Requests.Abstractions;
+using Messenger.Infrastructure.User;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Messenger.Data.Seeder;
 
 public class DbSeeder : IDbSeeder
 {
     private readonly Random _random = new();
 
-    public async Task Seed(IDbContext dbContext)
+    public async Task SeedAsync(IDbContext dbContext, IServiceProvider serviceProvider)
     {
-       
-        await SeedMessengerUsers(dbContext);
+        await SeedMessengerUsers(dbContext, serviceProvider);
         await dbContext.SaveEntitiesAsync();
     }
 
-    private async Task SeedMessengerUsers(IDbContext dbContext)
+    private async Task SeedMessengerUsers(IDbContext dbContext, IServiceProvider serviceProvider)
     {
         if (dbContext.MessengerUsers.Any())
             return;
-        var users = new List<MessengerUser>
+        
+        var userService = serviceProvider.GetRequiredService<UserManager<ApplicationUser>>();
+        
+        var appUsers = new (string name, Guid id)[] 
```

Comment:
В кортежах мы название элемента пишем с большой буквы
```suggestion
        var appUsers = new (string Name, Guid Id)[] 
```